### PR TITLE
[Draft ] Support 'route_metric4' for initscripts provider

### PR DESCRIPTION
**Pull request by liangwen12year**
Signed-off-by: Wen Liang <liangwen12year@gmail.com>

### CODE REVIEW

**Comment by liangwen12year** [comment]
[citest]

File: README.md [hunk 1]
```diff
@@ -526,7 +526,8 @@ The IP configuration supports the following options:
     [`ipv6.route-metric`](https://developer.gnome.org/NetworkManager/stable/nm-settings.html#nm-settings.property.ipv6.route-metric)
     properties, respectively. If specified, it determines the route metric for DHCP
     assigned routes and the default route, and thus the priority for multiple
-    interfaces.
+    interfaces. For `initscripts`, `route_metric4` sets the metric for the default
+    route and `route_metric6` is not supported.
```
Comment (richm) [inline README.md:530]: `route_metric4` only applies to the default route, not the `DHCP` route also?

Comment (liangwen12year) [reply README.md:530]: For nm provider, `route_metric4` applies to  all routes including the default route.
For initscripts provider, `route_metric4` only applies to the default route.

**Review by richm — APPROVED** [review]
only test failures are the known bond test failures

**Review by thom311 — APPROVED** [review]

**Comment by liangwen12year** [comment]
[citest]

**Review by ffmancera — APPROVED** [review]

**Pull request merged**